account_product_fiscal_classification/models/account_product_fiscal_classification.py

```diff
@@ -8,11 +7,11 @@
 
 
 class AccountProductFiscalClassification(models.Model):
-    """Fiscal Classification of customer and supplier taxes.
-    This classification is linked to a product to select a bundle of taxes
-     in one time."""
     _name = 'account.product.fiscal.classification'
     _inherit = 'account.product.fiscal.classification.model'
+    _description = 'Fiscal Classification of customer and supplier taxes.\
```

Comment:
AFAIK, this field is the "pretty name", displayed in the UI. So, it seems this one too long. What do you think ?